### DISCUSSION TITLE
Directus Interfaces in Extension Module

### DISCUSSION THREAD

**Discussion by bigdigitalSVN** [directus/directus#17730] [Archive]
Hi, I am creating my own module, that acts very similarly to Content/Collection Creation.

How do I access Directus-Made Layouts/Interfaces in a custom module? Is this possible to do?

I have tried looking into: https://docs.directus.io/extensions/modules.html#accessing-internal-systems
But not having much luck on using the actual Interfaces/Layouts.

**Comment by carwei** [comment]
I'm wondering the same thing. Can a built-in layout be used in a custom module?
